### DISCUSSION TITLE
How can we validate the multiple checkboxes using this module ?

### DISCUSSION THREAD

**Discussion by Ram2507** [npm/feedback#490] [General]
I have the following group of checkboxes
 <div className='cnp-form-group'>
            <label>Hobbies</label>
            <label><input type='checkbox' name='hobbies' value={this.state.hobbies} onChange={ this.formFieldHandler } />Playing</label>
            <label><input type='checkbox' name='hobbies' value={this.state.hobbies} onChange={ this.formFieldHandler } />Cooking</label>
            {this.validator.message('hobbies', this.state.hobbies, 'required')}
          </div>

**Comment by MylesBorins** [comment]
Hey @Ram2507 . This discussions form is for suggesting features for the npm registry, website, and CLI. We do not offer general technical support through this channel. You may want to try opening an issue over at https://github.community or on Stackoverflow. Best of luck!
